src/Domain/Orientation.hpp

```diff
@@ -15,10 +15,10 @@
 
 /*!
  * \ingroup DomainCreators
- * \brief Allows one to construct a custom Orientation between two Blocks.
- *
- * \usage The user provides `2*VolumeDim` Directions, encoding
- * the correspondence between the directions in each block.
+ * \brief A mapping from the logical coordinate axes of a host to the logical
+ * coordinate axes of a neighbor of the host.
+ * \usage Given a `size_t dimension`, a `Direction`, or a `SegmentId`, of the
+ * host, an `Orientation` will give the corresponding value in the neighbor.
```

Comment:
- Orientation -> OrientationMap
- mapped functions to operator()
- inverse -> inverse_map